test/test_sedcli_basic.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2018-2019 Intel Corporation
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+"""
+Tests end-user scenarios for sedcli binary.
+This suite needs to be run against Python 3.x and the sedcli binary must be
+in your $PATH.
+"""
+
+import os
+import sys
+import unittest
+import subprocess
+import random
+import string
+
+class TestSedcliBasic(unittest.TestCase):
+
+	NVME_DEV_PATH = '/dev/nvme0n1'

```

Comment:
The device path may change,  need not have to be nvme0n1 every time.
I think it would be a better idea to ask the user to enter the device path

Reply:
or maybe have a config file

Reply:
NVME_DEV_PATH = '/dev/nvme0n1' if the default setting that can be overriden when user specifies command line parameter. See below. I agree with you there are better ways to handle static configuration e.g. through config file

if __name__ == '__main__':
	if len(sys.argv) > 1:
		TestSedcliBasic.NVME_DEV_PATH = sys.argv.pop()